src/core.js

```diff
@@ -265,6 +265,7 @@ $.extend( $.validator, {
 		errorLabelContainer: $( [] ),
 		onsubmit: true,
 		ignore: ":hidden",
+		inputSelector: "input, select, textarea, [contenteditable]",
```

Comment:
Other settings dont use the suffix `Selector` we shouldnt use it either.

Reply:
Not sure yet what a good name should be for this setting.

Reply:
What about `targets`?
It aligns well with the internal method `validationTargetFor(...)`.